### PR TITLE
[REEF-656] Add csproj for Org.Apache.REEF.Examples.DriverRestart

### PR DESCRIPTION
**Pull request by afchung**
This addressed the issue by
  * Adding a project for Org.Apache.REEF.Examples.DriverRestart.

JIRA:
  [REEF-656](https://issues.apache.org/jira/browse/REEF-656)

### CODE REVIEW

**Comment by markusweimer** [comment]
This looks good to me, but I am no expert on the .NET build stuff. @jwang98052 Can you have a look?

File: lang/cs/Org.Apache.REEF.Examples.DriverRestart/App.config [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (jwang98052) [inline lang/cs/Org.Apache.REEF.Examples.DriverRestart/App.config]: Remove this file from the project and delete this file if you don't use it.

File: lang/cs/Org.Apache.REEF.Examples.DriverRestart/Org.Apache.REEF.Examples.DriverRestart.csproj [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="DriverRestart.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
```
Comment (jwang98052) [inline lang/cs/Org.Apache.REEF.Examples.DriverRestart/Org.Apache.REEF.Examples.DriverRestart.csproj]: can be removed.

**Comment by jwang98052** [comment]
Looks good. Just a few comments.

**Comment by afchung** [comment]
@jwang98052 Updated, please have another look. Thanks!

**Comment by jwang98052** [comment]
I will test and merge.

**Pull request closed without merging**